src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
+const COST_PER_DAY = 40;
+const MEDIUM_PERIOD = 3;
+const MEDIUM_PERIOD_DISCOUNT = 20;
+const LONG_PERIOD = 7;
+const LONG_PERIOD_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * COST_PER_DAY;
+
+  if (days >= LONG_PERIOD) {
+    return rentalCost - LONG_PERIOD_DISCOUNT;
+  }
+
+  if (days >= MEDIUM_PERIOD) {
+    return rentalCost - MEDIUM_PERIOD_DISCOUNT;
+  }
```

Comment:
The current logic applies only one discount based on the first condition met. If the intention is to apply both discounts cumulatively when the number of days is greater than or equal to 7, you should adjust the logic to account for both discounts. If the discounts are meant to be mutually exclusive, then this logic is correct.